### ISSUE TITLE
Will fasthax ever come back to 3DS?

### ISSUE THREAD

**Issue by Psionic8** [nedwill/fasthax#66]
Will you ever create a new fasthax for people who wanna get CFW on their 3ds or is it over now?

**Comment by Masamune3210** [comment]
It's dead, the exploit was patched in the latest update

**Comment by SkullMask** [comment]
Is there someone interested in trying to make another exploit in another way?

**Comment by Intege-rs** [comment]
This is an issue tracker not a place to beg for exploits, please go somewhere else.

**Comment by SkullMask** [comment]
You know i was just asking im fine i got my O3ds A9lh + Luna 11.3 i was just asking to try to help you little scumbag.

**Comment by Psionic8** [comment]
Yeah jeez what's your issue Jessie calm down

**Comment by Intege-rs** [comment]
Because this question has been asked multiple times and this isn't the place for it. When i said please go somewhere else I meant a place such as GBATemp, or reddit. It was completely unnecessary of you to call me a scumbag.

**Comment by lordsba** [comment]
sighax the next expoit will be able to downgrade 11.3 ti hack

**Comment by Intege-rs** [comment]
afaik you wont be able to use sighax without a previously hacked 3ds. https://gbatemp.net/threads/sighax-updates-and-discussion-thread.454821/ "You should better read this first" from a quick skim it appears to just be a way to get CFW without a9lh, not an exploit to downgrade or install cfw (without already having it or ability to get it). Please read the post first as what I have just stated could be completely wrong.

**Comment by Psionic8** [comment]
You're right except for you would need a hardmod otherwise. It is possible
without previous hacks if you are willing to hardmod your 3DS.
